content/400-reference/300-database-reference/01-database-features.mdx

```diff
@@ -127,7 +127,7 @@ The following table lists common MongoDB features and describes the level of sup
 | :---------------------------------------- | :------------------ | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | Embedded documents                        | Work in progress    | Embedded documents are not yet supported, but you can use the `Json` field while we work to provide support. See [GitHub issue #8272](https://github.com/prisma/prisma/issues/8272) to monitor progress. |
 | Transactions                              | ✔️                  |                                                                                                                                                                                                          |
-| Indexes                                   | ✔️ with caveats     | You cannot model indexes in the Prisma schema, but Prisma does not prevent you from creating indexes for your database.                                                                                  |
+| Indexes                                   | ✔️ with caveats     | Indexes can only be introspected if the field they refer to includes at least some data.                                                                                                                                                                                                         |
 | Autoincrementing IDs                      | No                  |                                                                                                                                                                                                          |
 | Compound IDs                              | No                  | MongoDB does not support composite IDs (`@@id`)                                                                                                                                                          |
```

Comment:
@Jan, does the phrase "composite IDs" seem possibly confusing here, since we do support "composite type"?? Should it it just stick with "compound ID"?

Reply:
I have not changed that wording and it is not related to MongoDB's index support.

("Compound ID" might be a term we use in other places. You would need to research that.)

Reply:
ping @tanberry

Reply:
Yeah, I'll look into that separately, good call.